joomla/joomla-cms
# [4.3] Fix wrong language string constant for category permission notes

Pull Request for Issue #39076 .

### Summary of Changes

Revert the replacement of the `JLIB_RULES_SETTING_NOTES_ITEM` language string override in file `com_categories.ini` by a string with the wrong name done with PR #35424 and remove the wrong string (which doesn't need a deprecation since never been in any stable or development release).

### Testing Instructions

Edit a category and go to the "Permissions" tab. Expand the notes and check what's written there.

### Actual result BEFORE applying this Pull Request

Standard language string is used in the permission notes.

### Expected result AFTER applying this Pull Request

Special language string for categories is used in the permission notes.

### Link to documentations
Please select:
